README.md

```diff
@@ -0,0 +1,109 @@
+# 🛸Project Goals🛸
+
+These are snatched from the original project (_OP_), with some adjustments and amendments.
+Even though the list is ordered, they are not (yet) ordered by priority, but merely to make each point adressable, as in `🛸2`.
+
+1. provide an idiomatic rust implementation for Google APIs, which includes _type safety_ and native _async_ operations.
+1. first-class documentation with cross-links and complete code-examples
+1. support all API features, including downloads and resumable uploads
+1. Convenient CLIs are provided on top of the API for use in shell scripts
+1. API and CLI generation can be customized easily to overcome issues with your particular API
+   * **Byron thinks that** we cannot assume to get it right for all APIs from the start unless we actually test everything ourselves. Thus if we enable people to help themselves, it will help us further down the line.
+1. Built-in debugging and tracing allows to understand what's going on, and what's wrong.
+   * **Byron thinks that** providing full output of requests and responses when using the CLI really helped. However, this time around there might be some more logging, using `tracing` or `log` at least. Here once again it becomes interesting to see if different systems can be supported, to allow people to tailor their experience based on their needs. `cargo features` could come a long way.
+1. The code we generate defines the standard for interacting with Google services via Rust.
+   * Google uses these crates! They are that good!
+1. The code base is made for accepting PRs and making contributions easy
+   * To stay relevant, people must contribute.
+   * The original authors won't stay around forever (see [GitPython](https://github.com/gitpython-developers/GitPython))
+1. _safety and resilience_ are built-in, allowing you to create highly available tools on top of it. For example, you can trigger retries for all operations that may temporarily fail, e.g. due to network outage.
+   * **Byron thinks that** this could be naturally supported by the async ecosystem, and thus helps slim down the delegate.
+
+# Learning from the past
+
+Let's keep in mind what worked and what didn't.
+
+## 🌈What worked well in _OP_🌈
+
+1. **`make` to track dependencies and drive tooling**
+   * Having built-in help was nice, and one go-to location to drive everything around the project
```

Comment:
I understand. To me it's about what's easier and faster to implement. For instance, it seems easy and clear to do something like `generate-lib drive && cd drive && cargo fmt`, and expose this via make, instead of having to look up Rust code.

The point I take for sure is to integrate better with `cargo`, so even _if_ make is used because it is indeed a good tool for the job, one can drive it via cargo as alternative entry point.

Something I really love is the unix philosophy of having multiple small tools that chain together via shell scripts.

However, I am totally able to be convinced to ditch `make` and go all Rust, maybe it is easier in the end. Maybe cargo can be taught how to run alternative programs, and what they depend on, maybe via `rerun-if-changed`. If will thoroughly think about how this could be achieved, because there is value in not needing and having to maintain `make`.

I am open, and probably sometimes stick to what I know and like (e.g. `make`) a little too much, kind of the opposite of your PTSD :D.